doc/langref.html.in

```diff
@@ -2529,16 +2529,16 @@ test "multidimensional arrays" {
 const std = @import("std");
 const expect = std.testing.expect;
 
-test "null terminated sentinel array" {
+test "0-terminated sentinel array" {
     const array = [_:0]u8 {1, 2, 3, 4};
 
     try expect(@TypeOf(array) == [4:0]u8);
     try expect(array.len == 4);
     try expect(array[4] == 0);
 }
 
-test "internal sentinels in sentinel array" {
-    // internal sentinels are not part of compile-time type, see Slices for variable-length sentinel-terminated arrays
+&#14;test "extra 0s in 0-terminated sentinel array" {
```

Comment:
This `&#14;` looks like it was added on accident.

Reply:
Oops!  Thanks for catching that.